### ISSUE TITLE
IndexError: string index out of range

### ISSUE THREAD

**Issue by Paul-hengChen** [titipata/science_concierge#24]
I'm a python beginner . I'm so sorry to ask this question ， but  I followed your Example usage of Science Concierge and got the error  as following 

```
' IndexError: string index out of range '
```

I tried many ways to fix it, but still can't solve it,  could you give me some direction?, please !

**Comment by titipata** [comment]
Hi @Paul-hengChen, thanks for the report! I have the same error too. I'm fixing it now and will let you know when it's done.

**Comment by titipata** [comment]
Wow, this is a problem of the stemmer that we use from `nltk`. The issue can be see on this Stack Overflow post: https://stackoverflow.com/questions/41517595/nltk-stemmer-string-index-out-of-range.

So this will be the bug for `nltk` version `3.2.2`. You can fix this by upgrading the library `pip install nltk --upgrade`. This will solve the problem of `PorterStemmer` bug that they have.

**Comment by titipata** [comment]
@Paul-hengChen, let me know if it fixes the issue. I upgraded my `nltk` version and the issue is resolved.

**Comment by Paul-hengChen** [comment]
I'm very thanks for your help!!!! The problem was resolved !!!!!  but I got another error

```
RuntimeWarning: Surrogate character u'\ud880' will be ignored. 
```

You might be using a narrow Python build. Just like this. Where I can improve? Thank you again for your help!

**Comment by titipata** [comment]
Ah, I didn't get that error. Can you check is it just a `RuntimeWarning` or is it an error? It might be just warning from `unidecode` library.

**Comment by Paul-hengChen** [comment]
Yeah , you are right ,  just warning  not error  !   thanks for your help !!!   you are so kind  :)  👍

**Comment by titipata** [comment]
No problem @Paul-hengChen! Glad to help.
